evap/results/exporters.py

```diff
@@ -101,16 +107,23 @@ def filter_text_and_heading_questions(questions):
         return filtered_questions
 
     @staticmethod
-    def filter_evaluations(semesters, evaluation_states, degrees, course_types, contributor, include_not_enough_voters):
+    def filter_evaluations(
+        semesters: Iterable[Semester],
+        evaluation_states: Any,
+        degree_ids: Iterable[int],
+        course_type_ids: CourseType,
```

Comment:
`course_type_ids` is plural, `CourseType` is singular. Should this be `Iterable[int]` or `Iterable[CourseType]`?